chatkit-core/src/test/kotlin/com/pusher/chatkit/users/UserSubscriptionTestUtil.kt

```diff
@@ -0,0 +1,4 @@
+package com.pusher.chatkit.users
```

Comment:
in the future rather no (still depending on approach taken for future work on the foundations), however for the changes here it would be good to have parser tests for other changed bits (joining/creating or fetching joinable rooms) and generally it looked better as something not specific to user sub, hence [done](21e4610536901eef5db6ca3e3fa6f6598e3d5201)